formats/tailwind/get-font-size.js

```diff
@@ -12,7 +12,7 @@ module.exports.getFontSize = (dictionary) => {
   return [...primaryFontSizes, ...secondaryFontSizes, ...textFontSizes].reduce(
     (acc, token) => ({
       ...acc,
-      [token.name.replace('typography-', '')]: [
+      [token.name.replace('typography-', '').replace('text-', '')]: [
```

Comment:
I think that the `text-` chunk should be only replaced for the aliases to preserve the backward compatibility. Please add also a TODO note to inform that we will need to replace it in future for all the `typography` tokens